src/toObject/toObject.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { toObject } from './toObject';
+
+describe('toObject function', () => {
+  const input = [['a', 1], ['b', 'c']];
+
+  test('is a pure function', () => {
+    toObject(x => `${x}`, x => x)(input);
+    expect(input).toMatchSnapshot();
+  });
+
+  test("[['a', 1], ['b', 'c']] |> toObject(key => key, value => value + value) === {a: 1, b: 'c'}", () => {
```

Comment:
`toObject(key => key, value => value + value)` functions not describes what the test did.

Reply:
Done